src/types/component.rs

```diff
@@ -166,22 +216,34 @@ pub enum TextStyle {
 #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum TextWrapMode {
+    /// Disable text wrapping. Text that does not fit inside the texture will be cut off.
     None,
+    /// Wraps at a glyph level.
     Glyph,
+    /// Preserve words when wrapping.
     Word,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum TextWeight {
+    /// Thin weight (100), the thinnest value.
```

Comment:
I'm not sure about it but I think `the thinnest value` and `the thickest value` sound a bit off. The font can be thin/thick but the value is small or large.

Reply:
those comments are copied from glyphone, but I agree this sound a bit "not yes" :D